osu.Game/Graphics/Containers/UserDimContainer.cs

```diff
@@ -90,7 +91,7 @@ protected virtual void UpdateVisuals()
             ContentDisplayed = ShowDimContent;
 
             dimContent.FadeTo(ContentDisplayed ? 1 : 0, BACKGROUND_FADE_DURATION, Easing.OutQuint);
-            dimContent.FadeColour(OsuColour.Gray(1 - (float)DimLevel), BACKGROUND_FADE_DURATION, Easing.OutQuint);
+            dimContent.FadeColour(IgnoreUserSettings.Value ? OsuColour.Gray(1 - (float)DimLevel) : Color4.White, BACKGROUND_FADE_DURATION, Easing.OutQuint);
```

Comment:
Is this line required? Shouldn't it be handled by the `DimLevel` one above?

Reply:
resolved in caa9286